pymatviz/ptable/plotly.py

```
@@ -1906,3 +1922,240 @@ def ptable_scatter_plotly(
                 trace.marker.colorbar = cbar_settings
 
     return fig
+
+
+def ptable_bars_plotly(
+    data: dict[str, dict[str, float]],
+    *,
+    colorscale: str = "RdBu",
+    log: bool = False,
+    colorbar: dict[str, Any] | Literal[False] | None = None,
```

Ruff (0.11.9)

Unused function argument: `colorbar`

(ARG001)